### DISCUSSION TITLE
Alert inside table

### DISCUSSION THREAD

**Discussion by AmreshSinha** [retypeapp/retype#267] [Q&A]
I wanted to add a alert component inside a table but it doesn't seems to work.
```
| Test  | Description |
| ------ | --------------- |
| Alert | !!! Note !!!   |
```

I guess its not working because for alert we need newlines
```
!!!
This is a alert!
!!!
```

But if I add newline the table will not render.
So, can I add a alert inside a table? If its not implemented yet will it get implemented in any future release? Or any workaround?

Any help will be appreciated!
Thanks!

**Comment by fabriciomurta** [comment]
Hi the answer is no, you can't place an alert within a table, we simply don't support an "inline alert" markdown, so it won't work as tables requires cells to be single lines.

We don't currently have plans to implement it at all but, of course, if there's enough community support for this feature, we will certainly consider it a possibility.

If there's any workaround I could think of at the moment, it would be, draw an alert as you need it, multi-line and all, build the page and copy-paste the alert's HTML in one line as the table cell. It will, of course, not follow changes in retype, say, if the generated code changes between releases. That will be static HTML. Once you step in the static HTML, Retype won't touch the code, so you're on your own. But that's, nevertheless, a possibility and the only "workaround" I can think about at the moment.

Upcoming Retype 2 will employ partials, which would basically allow you to define that "html block" as a partial and, say, pass the message value as a parameter to the partial. This will allow you to define your own alert component and reuse it everywhere in your docs. This makes the workaround better because, once you notice some styling change, instead of nitpicking every place you pasted that code, it would only be needed to change one instance to reflect throughout the whole project. So it would be much more feasible to rely on workarounds like this and general customizations!
